client/src/test/omtLinkProvider.test.ts

```diff
@@ -29,7 +33,9 @@ describe('OMTLinkProvider', () => {
         // the making of the links should be tested for the server
         // and the rendering of the links is the responsibility of vscode
         const requestMethod = "textDocument/documentLink";
+        console.log('doing test')
```

Comment:
Is this logging required for testing, or is it used only for debugging purposes? If required, maybe make the logging a bit more informative, otherwise you can remove it (doing test doesn't make sense to me :) )

Reply:
Nope, this is completely unnecessary for the tests, I forgot to remove it... :laughing:

- [ ] remove debug logging